test/mobile/test_bytecode.py

```diff
@@ -15,33 +25,231 @@
 #         increment = torch.ones([2, 4], dtype=torch.float64)
 #         return self.x + y + increment
 
-# output_model_path = pathlib.Path(tmpdirname, "script_module_v5.ptl")
+# output_model_path = Path(tmpdirname, "script_module_v5.ptl")
 # script_module = torch.jit.script(TestModule(1))
 # optimized_scripted_module = optimize_for_mobile(script_module)
 # exported_optimized_scripted_module = optimized_scripted_module._save_for_lite_interpreter(
 #   str(output_model_path))
 
+SCRIPT_MODULE_V4_path_BYTECODE_PKL = '''
+(4,
+ ('__torch__.*.TestModule.forward',
+  (('instructions',
+    (('STOREN', 1, 2),
+     ('DROPR', 1, 0),
+     ('LOADC', 0, 0),
+     ('LOADC', 1, 0),
+     ('MOVE', 2, 0),
+     ('OP', 0, 0),
+     ('LOADC', 1, 0),
+     ('OP', 1, 0),
+     ('RET', 0, 0))),
+   ('operators', (('aten::add', 'int'), ('aten::add', 'Scalar'))),
+   ('constants',
+    (torch._utils._rebuild_tensor_v2(pers.obj(('storage', torch.DoubleStorage, '0', 'cpu', 8),),
+       0,
+       (2, 4),
+       (4, 1),
+       False,
+       collections.OrderedDict()),
+     1)),
+   ('types', ()),
+   ('register_size', 2)),
+  (('arguments',
+    ((('name', 'self'),
+      ('type', '__torch__.*.TestModule'),
+      ('default_value', None)),
+     (('name', 'y'), ('type', 'int'), ('default_value', None)))),
+   ('returns',
+    ((('name', ''), ('type', 'Tensor'), ('default_value', None)),)))))
+        '''
+
+SCRIPT_MODULE_V5_path_BYTECODE_PKL = '''
+(5,
+ ('__torch__.*.TestModule.forward',
+  (('instructions',
+    (('STOREN', 1, 2),
+     ('DROPR', 1, 0),
+     ('LOADC', 0, 0),
+     ('LOADC', 1, 0),
+     ('MOVE', 2, 0),
+     ('OP', 0, 0),
+     ('LOADC', 1, 0),
+     ('OP', 1, 0),
+     ('RET', 0, 0))),
+   ('operators', (('aten::add', 'int'), ('aten::add', 'Scalar'))),
+   ('constants',
+    (torch._utils._rebuild_tensor_v2(pers.obj(('storage', torch.DoubleStorage, 'constants/0', 'cpu', 8),),
+       0,
+       (2, 4),
+       (4, 1),
+       False,
+       collections.OrderedDict()),
+     1)),
+   ('types', ()),
+   ('register_size', 2)),
+  (('arguments',
+    ((('name', 'self'),
+      ('type', '__torch__.*.TestModule'),
+      ('default_value', None)),
+     (('name', 'y'), ('type', 'int'), ('default_value', None)))),
+   ('returns',
+    ((('name', ''), ('type', 'Tensor'), ('default_value', None)),)))))
+        '''
+
+SCRIPT_MODULE_BYTECODE_PKL = {
+    4: {
+        "bytecode_pkl": SCRIPT_MODULE_V4_path_BYTECODE_PKL,
+        "model_name": "script_module_v4.ptl"},
+    5: {
+        "bytecode_pkl": SCRIPT_MODULE_V5_path_BYTECODE_PKL,
+        "model_name": "script_module_v5.ptl"},
+}
+
+MINIMUM_TO_VERSION = 4
```

Comment:
Maybe just add a comment here so people know how these are used.

Reply:
Comment is added.